packages/util/src/bytes.ts

```diff
@@ -25,19 +25,19 @@ export const bytesToUnprefixedHex = _bytesToUnprefixedHex
  * @throws If the input is not a valid 0x-prefixed hex string
  */
 export const hexToBytes = (hex: string) => {
-  if (!hex.startsWith('0x')) throw EthereumJSErrorWithoutCode('input string must be 0x prefixed')
+  if (!isHexString(hex)) throw EthereumJSErrorWithoutCode('input string must be 0x prefixed')
   return nobleH2B(padToEven(stripHexPrefix(hex)))
 }
 
 export const unprefixedHexToBytes = (hex: string) => {
-  if (hex.startsWith('0x')) throw EthereumJSErrorWithoutCode('input string cannot be 0x prefixed')
+  if (isHexString(hex)) throw EthereumJSErrorWithoutCode('input string cannot be 0x prefixed')
```

Comment:
The same here for `nobleH2B` as the other comment. I feel like the original `hex.startsWith('0x')` should stay here too.